source/topologies.txt

```diff
@@ -0,0 +1,36 @@
+============================
+Supported Cluster Topologies
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:ref:`c2c-mongosync` supports two cluster topologies: 
+
+- :ref:`Replica sets <replication>`
+
+- :ref:`Sharded clusters <sharded-cluster>`
+
+To sync a replica set to a replica set, see the :ref:`Quick Start Guide
+<c2c-quickstart>`.
+
+To sync a replica set to a sharded cluster, see :ref:`c2c-rs-to-sharded`.
+
+To sync a sharded cluster to a sharded cluster, see :ref:`c2c-sharded-clusters`.
+
+.. note::
```

Comment:
**Suggestion**

I might remove the note here and make a "Requirements" header and move the current note bullets here to that section before the "Get Started" header. Typcially we reserve the notes for supplemental information, but these two points seem like hard requirements which should be highlighted at the top of the page.

Reply:
Ok, made a "Requirements" section with the note info and the topology options. What do you think?